### ISSUE TITLE
Update cerulean mp profile

### ISSUE THREAD

**Issue by xyzerobtc** [zeroauthority-dao/za-website#18]
Scope: 
Once user connects wallet they will see the banner with STX BTC balance. 
Anons can update their profile with information. 
discord id, twitter id, type of services they will provide, stx address, btc address 
Users will be able to see their BNS. 

Amount: 
tbd

**Comment by xyzerobtc** [comment]
@SushilBro 
If these two items are too difficult to achieve, please let us know here and we will put it on the backlog. 
- Users will be able to see their BNS.
- User will be able to see in progress work or completed work by the 6 ending characters of the stx tx id

**Comment by SushilBro** [comment]
Certainly! Instead of saying that the items are too difficult to achieve, I will do my best to solve these problems. Please allow me some time to work on them, and I will keep you updated on my progress. Thank you for your patience.

**Comment by xyzerobtc** [comment]
Great mindset @SushilBro -- that is so true! 
We are making progress!
<img width="1091" alt="Screen_Shot_2023-03-06_at_3 29 03_AM" src="https://user-images.githubusercontent.com/16929732/223896996-515bf2dd-c098-4ae2-a4c8-b026c73f9bb0.png">
